### PR TITLE
Added notice that batching doesn't work with lightmaps

### PR DESCRIPTION
**Pull request by yaustar**
On feedback of forum: https://forum.playcanvas.com/t/lightmapping-simply-not-working/19268/10?u=yaustar

Have I got the information here correct?

### CODE REVIEW

File: content/en/user-manual/graphics/lighting/lightmapping.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ PlayCanvas offers two ways to use lightmaps in your scene: **External lightmap g
 
 This page gives details and best practices on rendering lightmaps from external tools.
 
+<div class="alert-info">
```
Comment (mvaligursky) [inline content/en/user-manual/graphics/lighting/lightmapping.md:20]: This limitation does not apply to lightmaps baked in other tools .. as if those bake lightmaps to atlas as they normally do, batching would work. So I'd suggest to remove this part.

File: content/en/user-manual/graphics/lighting/runtime-lightmaps.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ The advantages are so:
 
 The disadvantage of using runtime lightmap generation is that currently we do not support baking global illumination, ambient occlusion or some of the other advanced features of a specialized baking tools.
 
+<div class="alert-info">
```
Comment (mvaligursky) [inline content/en/user-manual/graphics/lighting/runtime-lightmaps.md:28]: I'd change both runtime lightmaps and batching to something along: The use of batching is not compatible with the runtime lightmaps due to each lightmapped object requiring its own unique lightmap.

File: content/en/user-manual/graphics/lighting/lightmapping.md (hunk 1 above)
Comment (yaustar) [reply content/en/user-manual/graphics/lighting/lightmapping.md:20]: When you say bake, do you mean as part of the diffuse or as a separate texture for the lightmap section of the material or either?
image

File: content/en/user-manual/graphics/lighting/runtime-lightmaps.md (hunk 2 above)
Comment (yaustar) [reply content/en/user-manual/graphics/lighting/runtime-lightmaps.md:28]: Why does having a unique lightmap affect batching though?

File: content/en/user-manual/graphics/lighting/lightmapping.md (hunk 1 above)
Comment (mvaligursky) [reply content/en/user-manual/graphics/lighting/lightmapping.md:20]: either. External lightmaps either stored in diffuse or lightmap slot of materials work with batching - as material batching works. Internally baked lightmaps are not stored in material, but in meshInstances .. and that's the reason batching does not work. And the reason they're stored in meshInstances is that each meshInstance has different lightmap due to not having atlas, and so they are different textures. We need to implement lightmap atlasing which is on my list in the next few months.

File: content/en/user-manual/graphics/lighting/runtime-lightmaps.md (hunk 2 above)
Comment (mvaligursky) [reply content/en/user-manual/graphics/lighting/runtime-lightmaps.md:28]: well it's the same as having two meshes, and each uses different diffuse texture. They cannot be batches, as the resulting batch can only use single diffuse texture. If they're on an atlas, then can be batched as they'd use the same texture (the atlas one)

Comment (yaustar) [reply content/en/user-manual/graphics/lighting/runtime-lightmaps.md:28]: Gotcah, thanks!

**Comment by yaustar** [comment]
Updated with Martin's suggestions

**Review by mvaligursky — APPROVED** [review]

**Pull request merged**